src/core/client/framework/schema/custom.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * TODO: (cvle) This file is a workaround to have Relay compatible enum types.
+ * This should be generated by `graphql-schema-typescript`.
+ */
+
+import { RelayEnumLiteral } from "../types";
+import {
+  GQLCOMMENT_FLAG_DETECTED_REASON,
+  GQLCOMMENT_FLAG_REASON,
+  GQLCOMMENT_FLAG_REPORTED_REASON,
+  GQLCOMMENT_SORT,
+  GQLCOMMENT_STATUS,
+  GQLLOCALES,
+  GQLMODERATION_MODE,
+  GQLSTORY_STATUS,
+  GQLUSER_AUTH_CONDITIONS,
+  GQLUSER_ROLE,
+} from "./__generated__/types";
+
+export type GQLUSER_ROLE_RL = RelayEnumLiteral<typeof GQLUSER_ROLE>;
```

Comment:
The `_RL` suffix is a bit confusing, perhaps instead:

```suggestion
export type RLGQLUSER_ROLE = RelayEnumLiteral<typeof GQLUSER_ROLE>;
```

To match the `GQL` prefix?

Reply:
Deferring fix to a later date when we can work more with `graphql-schema-typescript`.